webpack/assets/javascripts/foreman_navigation.js

```diff
@@ -20,3 +20,15 @@ export const changeOrganization = org => {
 export const changeActive = active => {
   store.dispatch(LayoutActions.changeActiveMenu({ title: active }));
 };
+
+export function showContent(layout, unsubscribe) {
```

Comment:
I'm wondering, shouldn't this workaround suppose to live in the legacy code?

Reply:
we gain advantages if this callback live under webpack, especially for testing.